src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -5184,7 +5245,17 @@ void InstrumentClipView::graphicsRoutine() {
 	uint8_t nonMutedColour = clip->getCurrentlyRecordingLinearly() ? 2 : 0;
 	for (int32_t yDisplay = 0; yDisplay < kDisplayHeight; yDisplay++) {
 		int32_t noteRowIndex;
-		NoteRow* noteRow = clip->getNoteRowOnScreen(yDisplay, currentSong, &noteRowIndex);
+		NoteRow* noteRow = nullptr;
+		// if we're in the automation view note editor, then we're only looking at one note row at a time
+		// so we want to render a single note row's playhead across all note rows
+		if (getRootUI() == &automationView && automationView.inNoteEditor()) {
+			noteRow = clip->getNoteRowOnScreen(lastAuditionedYDisplay, currentSong, &noteRowIndex);
```

Comment:
this seems inefficient - probably put this outside the loop so we don't repeatedly grab the noterow on screen while updating